packages/core/src/external/mpi/normalize-patient.ts

```diff
@@ -0,0 +1,164 @@
+import { PatientDataMPI, Address } from "./patient-incoming-schema";
+
+// Define default values for each field
+const defaultValues = {
```

Comment:
I dont understand the purpose of the defaultValues

Reply:
The idea is that sometimes people will insert values to fill in fields that are not real values, and they pollute the patient matching system, and you want to filter them out. If one customer submits a patient who they dont know the address for as 123 Main St, and then we are recieving requests with someone doing a similar thing, the patients will look like they are matching.

Reply:
I see, the issue with this is `123 Main St` is an actual street name

Reply:
yah not sure what to do here. Maybe just remove the address value from the default list?

Reply:
will comment out the call to `handleDefaultValues`. Especially if we aren't sure that this is even currently happening with any of our customers.

Reply:
I think this is a valid concern and I would avoid losing the opportunity to do something about it.

I suggest we take create a ticket to move this to validation and share on #engineering for vis.